lib/network/http2.go

```diff
@@ -152,6 +152,16 @@ func (t *HTTP2Network) setNotReadyHandler() {
 	t.server.Handler = HTTP2Log15Handler{log: t.log, handler: t.router}
 }
 
+func (t *HTTP2Network) AddMiddleware(routerName string, mws ...mux.MiddlewareFunc) {
+	r, ok := t.routers[routerName]
+	if !ok {
+		// ignore it
```

Comment:
Instead of ignoring it, I think returning error is safe.

Reply:
Ok, Good point :-)